### ISSUE TITLE
The Microsoft narrator focus does not sync with the keyboard focus on the sub menu item.

### ISSUE THREAD

**Issue by Deena-the-coder** [microsoft/fluentui#21075]
### Environment Information

- **Package version(s)**: fluent ui 7.176.2
- **Browser and OS versions**: Windows 10 Version 21H1

### Describe the issue:
We are using Calendar component as a sub menu item in ContextualMenu.There are multiple menu items available and the keyboard focus and narrator focus works in sync while navigating through the menu items.But when w etry navigating to the sub menu item from a menu item,only the keyboard focus shifts and the narrator focus stays with the parent menu item itslef.Because of this, the screen reader users will get impacted.

### Please provide a reproduction of the issue in a codepen:
1. Open 'https://beta.to-do.microsoft.com/' link and sign in with valid credentials​
2. Navigate to and select a list from the left navigation pane
3. Navigate to a task and invoke the details view of that task​
4. In the details pane, navigate to Remind me option.
5. In The reminder popup, navigate and select Pick a date & time option.
6. Observe keyboard focus and Narrator focus.

#### Actual behavior:

Upon selecting pick a date & time option, Narrator focus remain on the menu option. Keyboard focus will be on Next month button of Pick a date calendar dialog.

#### Expected behavior:

Upon selecting pick a date & time option, Narrator focus should move along with keyboard focus and should announce the focused element details.

**Comment by Deena-the-coder** [comment]
@layershifter  : Just wanted to check if there is any update on this?

**Comment by gouttierre** [comment]
@Deena-the-coder - I’ve added this to our contributor backlog for review for later investigation/ contribution but it’s low on our priority list as we’re likely not going to get to it for a while as we’re currently focused on larger coherence work items and enhancements to our performance.
